src/java.base/aix/classes/jdk/internal/loader/ClassLoaderHelper.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2012, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

New file.  Copyright header start year should be 2024.